elastic/kibana
# feat: 🎸 connect dasdhboard telemetry to persistable state

## Summary

Connects Dashboard telemetry collector to Embeddables service (`PersistableState` interface). This way telemetry is collected from the following chain:

```
Dashboard -> Embeddable -> UI actions -> Drilldowns
```

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers


### How to test

1. Set up a dashboard, configure your dashboard, for example, add a drill-down.
2. Go to "Stack Management" 👉 "Advanced Settings" 👉 at the very bottom click on link that opens flyout with collected telemetry data.
3. Look for `dashboard` data, there should be new `embeddable` key with embeddable service telemetry.
